### PR TITLE
Branch docs user guide

### LINKED ISSUE #153: [PE-D] Glossary is missing in UG
UG does not have a glossary to explain technical terms

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: yanchenglee98/ped#8

### PR DESCRIPTION
**Pull request by yingqi0607**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -112,7 +116,7 @@ Here are some commands that you might find it handy when using TR4CKER.
 
 ### 3.1.1. Viewing help : `help`
 If you cannot recall the command you need, you can enter the help command in any panel.
-A help window will pop out with a link to this User Guide.
+You will be switched to help tab and a help window will pop out with a link to this User Guide.
```
Comment (ruilingk) [inline docs/UserGuide.md]: phrasing abit awkward, maybe can say `TR4CKER will switch to help tab and a help window will pop out with a link to this User Guide.`?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -226,7 +231,7 @@ Examples:
 
 ### 3.2.4. Listing all tasks: `list`
 
-Displays a list of all tasks names in TR4CKER.
+Displays a list of all pending tasks names in TR4CKER.
```
Comment (ruilingk) [inline docs/UserGuide.md]: i think can be more user-centric, so `If you would like to view a list of tasks, ...` like this?

**Review by ruilingk — COMMENTED** [review]
minor tweaks and it lgtm

File: docs/UserGuide.md [hunk 3]
```diff
@@ -372,31 +377,66 @@ If you would like to switch to Daily tab without clicking on the Daily tab butto
...
 
 ![todo result](images/dailyTab_todo1After.png)
 Figure 9: Daily tab after executing `todo 1`
 
-Example:
-* `list` followed by `todo 1` adds the first task in the displayed task list into daily todo list.
+### 3.3.3 Editing a daily todo task:
+You can edit details of a daily todo task using the `edit` command that is used for pending task.
+
+* When you edit the task name or deadline of a pending task, if this task has been added into daily todo list using the `todo` command, the
```
Comment (ruilingk) [inline docs/UserGuide.md:413]: want to consider adding screenshots?

File: docs/UserGuide.md (hunk 1 above)
Comment (ethan-l-m-e) [reply docs/UserGuide.md]: Is it even a tab? Something like a drop down bar?

**Review by hanweic53 — APPROVED** [review]
Looks good to merge.

**Pull request merged**